lsp/src/test/java/com/salesforce/slds/lsp/RecommendationTests.java

```diff
@@ -161,18 +161,42 @@ void invalidDesignToken() {
             CodeAction action = getCodeAction(RANGE, diagnosticResults);
             assertAction(action, "Remove design token 'testing'", "");
         }
+    }
 
+    @Nested
+    class MOBILE {
         @Test
-        void markupClassRecommendation() {
```

Comment:
Do we remove this test class?

Reply:
Oops, accidentally removed it. Thanks for catching!